.github/workflows/cancel_pr_runs.yml

```
@@ -0,0 +1,33 @@
+          - LintCode
+          - TestCode
+          - TestCodeMinDeps
+          - TestDocBuild
+          - TestFreeThreaded
+          - TestWasm
+    concurrency:
+      group: ${{ matrix.workflow }}-${{ github.event.pull_request.number }}
+      cancel-in-progress: true
+    steps:
+      - name: Confirm cancellation
+        run: echo "Canceled obsolete ${{ matrix.workflow }} runs for PR #${{ github.event.pull_request.number }}."
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1009:info:1:1: The mentioned syntax error was in this simple command

(shellcheck)